storage/migrations/0009_oasis_3_emerald_init.up.sql

```diff
@@ -16,10 +16,82 @@ CREATE TABLE IF NOT EXISTS oasis_3.emerald_rounds
   messages_hash    TEXT NOT NULL,
   in_messages_hash TEXT NOT NULL,
 
+  -- TODO: These should also be NOT NULL, but they're populated separately.
+  num_transactions INTEGER,
+  gas_used         BIGINT,
+  size             INTEGER,
+
   -- Arbitrary additional data.
   extra_data JSON
 );
 
+CREATE INDEX ix_emerald_rounds_block_hash ON oasis_3.emerald_rounds USING hash (block_hash);
+
+CREATE TABLE IF NOT EXISTS oasis_3.emerald_transactions
+(
+  round       BIGINT NOT NULL,
+  tx_index    INTEGER NOT NULL,
+  tx_hash     TEXT NOT NULL,
+  tx_eth_hash TEXT,
+  -- raw is hex(cbor(UnverifiedTransaction)). If you're unable to get a copy
+  -- of the transaction from the node itself, parse from here. Remove this if
+  -- we later store sufficiently detailed data in other columns or if we turn
+  -- out to be able to get a copy of the transaction elsewhere.
+  raw         TEXT NOT NULL,
```

Comment:
Let's use `BYTEA` unless there's a very strong reason for hex encoding. It takes up half as much space, requires less processing (ok, that's negligible), and the type better documents the nature of the data.

Reply:
gonna merge and address this separately